traffic_ops/app/db/seeds.sql

```diff
@@ -81,6 +87,7 @@ insert into tenant (name, active, parent_id) values ('unassigned', true, (select
 -- out of the box, only 3 roles are defined. Other roles can be created by the admin as needed.
 insert into role (name, description, priv_level) values ('admin', 'Has access to everything.', 30) ON CONFLICT (name) DO NOTHING;
 insert into role (name, description, priv_level) values ('operations', 'Has all reads and most write capabilities', 20) ON CONFLICT (name) DO NOTHING;
+insert into role (name, description, priv_level) values ('portal', 'Portal User', 15) ON CONFLICT (name) DO NOTHING;
```

Comment:
i think you can get rid of this one. see the comment from line 87

`-- out of the box, only 3 roles are defined. Other roles can be created by the admin as needed.` <-- that comment should really say 4 roles.

Reply:
can you change that comment to say "4"?

Reply:
I've changed it to 5, since there appear to be 5 roles defined out-of-the-box:

* admin
* operations
* portal
* read-only
* disallowed

(not sure where "steering" and "extension" come into play? Maybe just unenforced convention?)

Reply:
the idea here was that "out of the box" you get 4 standard roles (admin, ops, r/o and disallowed). portal needs to die as it makes no sense. it's like having a "traffic ops" role to use traffic ops...it was poorly named and does not have any semantic meaning.

Reply:
also, if you look at patches.sql, you'll see that "capabilities" are added to the federation, portal and steering roles (if they exist). this was to ensure no loss of functionality when the permissions system is migrated away from priv_level to roles/capabilities based. but out of the box we made a decision to just support the 4 standard roles. I'm pretty sure there was a dev email about this. I can see if I can dig it up.

Reply:
Done